core/plugins/jdo/datanucleus-5/src/test/java/org/apache/isis/runtime/persistence/objectstore/transaction/PojoAdapterBuilder.java

```diff
@@ -175,28 +165,18 @@ public PojoAdapterBuilder with(SpecificationLoader specificationLoader) {
     }
 
     public PojoAdapterBuilder with(AuthenticationSession authenticationSession) {
-        this.authenticationSession = authenticationSession;
```

Comment:
was it intentional to remove this line?  If so, what purpose does the method now have?

Reply:
unused fields removed, but didn't remove unused methods, fixing that